### ISSUE TITLE
core.authenticate docs discrepancy

### ISSUE THREAD

**Issue by ArtskydJ** [coding-in-the-wild/just-login-core#2]
@TehShrike The docs say that `core.authenticate` does:
```js
cb(err, contactAddress)
```
Right now it is doing:
```js
cb(err, {contactAddress, sessionId} )
```

Which way should it be?

**Comment by TehShrike** [comment]
Huh.  The latter seems fine to me I suppose.  I'm surprised that hasn't come up before.  Maybe that part of the documentation was just really old?

**Comment by ArtskydJ** [comment]
Fixed by 42d2eca.
